packages/core/base/src/utils/i18n.ts

```diff
@@ -0,0 +1,42 @@
+import {Locale} from "../models/types";
+
+const enUS = {
+    crossmintPayButtonService: {
+        CONNECTING: "Connecting...",
+        BUY_WITH_ETH: "Buy with ETH",
+        BUY_WITH_SOL: "Buy with SOL",
+        BUY_WITH_CREDIT_CARD: "Buy with credit card",
+    },
+    crossmintStatusButtonService: {
+        INVALID: "Invalid clientId",
+        WAITING_SUBMISSION: "Click here to setup Crossmint",
+        PENDING: "Your application is under review",
+        ACCEPTED: "You're good to go!",
+        REJECTED: "Your application was rejected"
+    },
+}
+
+const esES = {
+    crossmintPayButtonService: {
+        CONNECTING: "Conectando...",
+        BUY_WITH_ETH: "Comprar con ETH",
+        BUY_WITH_SOL: "Comprar con SOL",
+        BUY_WITH_CREDIT_CARD: "Comprar con tarjeta de crédito",
+    },
+    crossmintStatusButtonService: {
+        INVALID: "clientId inválido",
+        WAITING_SUBMISSION: "Haga clic aquí para configurar Crossmint",
+        PENDING: "Su solicitud está en revisión",
+        ACCEPTED: "¡Está listo para usar Crossmint!",
+        REJECTED: "Su solicitud fue rechazada"
+    }
+}
+
+const localeMap = {
+    "en-US": enUS,
+    "es-ES": esES,
+}
+export function t(wordingKey: string, locale: Locale) {
```

Comment:
`wordingKey` being a string isnt great

Can we either make it a type literal, or object

Reply:
Wanted to keep the "object.path.format", cause its kind of a standard, so found a way to make it work!

https://javascript.plainenglish.io/advanced-typescript-type-level-nested-object-paths-7f3d8901f29a